### DISCUSSION TITLE
In saved msgpack, rot_cw should be 3x3 but isn't.

### DISCUSSION THREAD

**Discussion by scimad** [stella-cv/stella_vslam#290] [Q&A] [answered]
## Describe the bug
While reading msgpack saved from videoslam example (video.mp4 from aist database), the msgpack containes rot_cw data incorrectly.

Here's my debug window.

![image](https://user-images.githubusercontent.com/11834788/165059023-698eeffe-986c-4489-93f4-283f8c9bcf78.png)

Am I missing something or is it a bug?

**Comment by ymd-stella** [comment, marked as answer]
It is stored as a quaternion.
